### PR TITLE
nagios-plugins are made by the Nagios Plugins Project

### PR DESCRIPTION
**Pull request by MisterMountain**
In this Pull Request, i made clear that the nagios-plugins packages on EL-OS are from the Nagios Plugins Project and not by the Monitoring Plugins Project.

### CODE REVIEW

File: doc/02-installation.md [hunk 1]
```diff
@@ -340,6 +340,8 @@ Without plugins Icinga 2 does not know how to check external services. The
 an extensive set of plugins which can be used with Icinga 2 to check whether
 services are working properly.
 
+The nagios-plugins Packages are provided by the [Nagios Plugins Project](http://nagios-plugins.org/).
```
Comment (krishnacx) [inline doc/02-installation.md:343]: *Packages -> packages

**Comment by bobapple** [comment]
@cla-bot check

File: doc/02-installation.md (hunk 1 above)
Comment (MisterMountain) [reply doc/02-installation.md]: Thx, fixed this with another commit

**Review by Al2Klimov — APPROVED** [review]

**Comment by Al2Klimov** [comment]
## TODO on merge

Select the squash option.

**Pull request closed without merging**